paint/PAINT_Screen.cpp

```diff
@@ -50,6 +50,18 @@ Screen::Screen(SDL_Renderer* renderer, const gfx::Rectangle& rect, const char* n
 			auto button = std::make_shared<ToggleButton>(renderer, buttonRect, "drawButton", "button_toggle_draw.png", toggleDraw);
 			toolbox->addChild(button);
 		}
+		else if (i == 1) {
```

Comment:
This `if (i == ...)` stuff looks really klunky and isn't very maintainable. Better to have an array of locally defined structures called (say) `ButtonInfo` that you can loop through.
**SHOULD**

Reply:
done